pandas/tests/frame/indexing/test_setitem.py

```diff
@@ -298,6 +298,14 @@ def test_iloc_setitem_bool_indexer(self, klass):
         expected = DataFrame({"flag": ["x", "y", "z"], "value": [2, 3, 4]})
         tm.assert_frame_equal(df, expected)
 
+    def test_setitem_iloc_pure_position_based(self):
+        # GH: 22046
+        df1 = DataFrame({"a2": [11, 12, 13], "b2": [14, 15, 16]})
+        df2 = DataFrame({"a": [1, 2, 3], "b": [4, 5, 6], "c": [7, 8, 9]})
+        df2.iloc[:, [1]] = df1.iloc[:, [0]]
```

Comment:
the test is for `iloc.__setitem__`, right?  test goes in tests.indexing.test_iloc

there's a fairly-new doc for this in doc/source/development/test_writing.rst

Reply:
Yes, moved it.

That is my bad then. Have just checked the 1.1.4 docs last week and could not find anything. Did not think about the repository itself :(